cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -101,7 +101,7 @@ metadata:
   # should be in the namespace of the operator
   namespace: rook-ceph # namespace:operator
 data:
-  # The logging level for the operator: INFO | DEBUG
+  # The logging level for the operator: CRITICAL | ERROR | WARNING | NOTICE | INFO | DEBUG
```

Comment:
I'm not aware of Rook using `critical` or `notice` anywhere. Shall we remove those options from the docs if there is no real application?

Reply:
That sounds reasonable.